src/main/java/edu/usf/cutr/gtfsrtvalidator/api/resource/GtfsFeed.java

```diff
@@ -123,7 +123,10 @@ public Response postGtfsFeed(@FormParam("gtfsurl") String gtfsFeedUrl) throws Js
                             + "WHERE gtfsUrl = '"+gtfsFeedUrl+"'").uniqueResult();
         GTFSDB.commitAndCloseSession(session);
         ObjectMapper mapper = new ObjectMapper();
-        downloadGtfsFeed(saveFilePath, connection);
+        Response.Status response = downloadGtfsFeed(saveFilePath, connection);
+        if(response == null)
```

Comment:
Please use braces for all If statements, even if they are one line:
`if (response == null) {`